polaris-react/src/components/Box/Box.tsx

```diff
@@ -207,24 +187,23 @@ export const Box = forwardRef<HTMLElement, BoxProps>(
     },
     ref,
   ) => {
+    const borderStyleValue = borderStyle
+      ? borderStyle
+      : (borderColor || borderWidth) ?? 'solid';
```

Comment:
Am I misreading this or does it actually do nothing because you already have the default set. With the default you should just be able to use borderStyle directly, no need for borderStyleValue

Reply:
I am going to remove the default value. I don't want every box rendering the CSS for `borderStyle` unless it is applying a border.